packages/config/config/devices/0x027a/templates/zooz_template.json

```
@@ -190,6 +224,29 @@
 		"unit": "%",
 		"defaultValue": 99
 	},
+	"double_tap_enable": {
+		"label": "Enable Double-Tap behavior",
```

```suggestion
		"label": "Double-Tap Behavior",
```
we try to avoid Enable/Disable in the param label